pkg/replication/replication.go

```diff
@@ -81,9 +87,10 @@ func (opts Options) Tags() []Tag {
 // Config - replication configuration specified in
 // https://docs.aws.amazon.com/AmazonS3/latest/dev/replication-add-config.html
 type Config struct {
-	XMLName xml.Name `xml:"ReplicationConfiguration" json:"-"`
-	Rules   []Rule   `xml:"Rule" json:"Rules"`
-	Role    string   `xml:"Role" json:"Role"`
+	XMLName       xml.Name `xml:"ReplicationConfiguration" json:"-"`
+	Rules         []Rule   `xml:"Rule" json:"Rules"`
+	Role          string   `xml:"Role" json:"Role"`
+	ReplicateSync bool     `xml:"ReplicateSync" json:"ReplicateSync"`
```

Comment:
Both aws sdk go and aws cli seem to have validations to reject any fields other than Rules/Role - so this change will break the sdk compatibility

Reply:
> Both aws sdk go and aws cli seem to have validations to reject any fields other than Rules/Role - so this change will break the sdk compatibility

maybe let's hide this field then (omitempty), I guess it will break only when someone enables this flag